cmd/dispatcher/main.go

```diff
@@ -42,7 +42,7 @@ type envConfig struct {
 	Requeue bool `envconfig:"REQUEUE" default:"false" required:"true"`
 
 	// Number of concurrent messages in flight
-	PrefetchCount int           `envconfig:"PREFETCH_COUNT" default:"10" required:"false"`
+	PrefetchCount int           `envconfig:"PREFETCH_COUNT" default:"1" required:"false"`
```

Comment:
I don't think we should change the default, we just introduced it

Reply:
Does this answer it? https://github.com/knative-sandbox/eventing-rabbitmq/pull/522#issuecomment-981816474